client/config.c

```diff
@@ -440,3 +456,119 @@ _TDNFConfigReadPluginSettings(
 error:
     goto cleanup;
 }
+
+uint32_t
+TDNFReadMinVersionsFiles(
+    char *pszDir,
+    char ***pppszMinVersions
+    )
+{
+    uint32_t dwError = 0;
+    DIR *pDir = NULL;
+    struct dirent *pEnt = NULL;
+    char *pszFile = NULL;
+    char **ppszNewMinVersions = NULL;
+    char ***pppszArrayList = NULL;
+    int nFileCount = 0;
+    int i, j, k;
+    int nLineCount = 0;
+    int nTmp;
+
```

Comment:
pls check for the arguments if they are valid.

Reply:
Done.